geo-traits/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "geo-traits"
+version = "0.1.0"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/georust/geo"
+documentation = "https://docs.rs/geo-traits/"
+readme = "../README.md"
+keywords = ["gis", "geo", "geography", "geospatial"]
+description = "Geospatial traits"
+rust-version = "1.65"
```

Comment:
Bumped this to 1.65 for this crate because the traits use GATs, which came in 1.65